README.md

```diff
@@ -1,6 +1,6 @@
 # Web Content Accessibility Guidelines Summary
 
-This document will help you quickly get up to speed with Web Content Accessibility Guidelines 2.1 (WCAG), and avoid common accessibility-related mistakes.
+This document will help you quickly get up to speed with Web Content Accessibility Guidelines 2.1 (WCAG), avoid common accessibility-related mistakes, and will provide code examples where possible.
```

Comment:
Happy to revert this but wanted to highlight that there is content here for developers too.

Wondering whether we need to make it clear _which_ detail pages have code examples, and for what platform.

Reply:
I think that we could add a separate, very short paragraph (rather than extend an existing sentence which is already quite long).

It could read something like: "Click on each guideline to see more details and code examples".

Of course we won't have code examples for each. But we could add "Not applicable" or "Work in progress" when we don't have a code example. Or just not have them where it doesn't make sense (eg Contrast).

Reply:
I'd like to indicate somehow which do have code samples if we can, not sure how that should look yet though

Reply:
There might be a way to do this without thwarting other outcomes we want. Very happy to explore.

But I'm afraid of adding anything that's not essential, because I'm imagining that it'd make the document too complex. Eg. would we need to indicate when there are code examples for iOS vs Android vs Web? What's the point of know that there are code samples if they are not relevant to you? This is just what I'm imagining would happen.

#### Suggested process for making difficult design decisions

When trying to make difficult design decisions, I'd like to start with outcomes and actual user problems:

##### Question 1: (about outcomes):
**What negative outcome are you imagining will happen if we don't?**

##### Question 2 (about checking what happens in reality):
**Are these negative outcomes actually happening in user testing / usage situations?**
If not, it's generally better not to optimise all things proactively, because everything is a trade-off.

##### Question 3 (about trade-offs):
**How bad is it when it happens?** (in relation to other negative outcomes that we might get by adding more info on the overview).

This is how we figure out what trade-offs we should make.

#### Suggested next steps

Let's discuss / design / user test this together. Based on this it'll be easier to know what to do / what not to do.